backend/src/core/entity/ReimbursementPlan.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * Describe a list of transactions to perform to reestablish the group balance
+ */
+export class ReimbursementPlan {
+  /** Id of the group from which this balance was calculated */
+  public readonly groupId: number;
+
+  /**
+   * @example amount = this.reimbursementPerMemberId.get(payerId).get(payeeId);
+   */
+  public reimbursementPerMemberId: Map<number, Map<number, number>>;
```

Comment:
Est-ce qu'on veut pas définir une interface intermédiaire pour être plus lisible (ReimbursementPlanPerMember) ?

Reply:
Pas trop chaud pour l'instant, c'est écrit que 2 fois pour l'instant et ça cacherait le type un peu ^^

De plus, c'est aussi un peu une implémentation technique "cachée" de l'entitée ^^ Je voulais rajouter des méthodes pour récupérer certaines données plus facilement, mais j'ai décidé de ne pas le faire pour pas prendre le taff des gens qui feront les implémentations ^^

J'avais pensé aux méthodes :
*  `getAllReimbursements(): Array<{from: number, to; number, amount: number}>`
* `getMemberReimbursements(member: GroupMember): Array<{to; number, amount: number}>`
* `getReimbursementAmount(payer: GroupMember, payee: GroupMember): amount | null` (pas trop trop sûr de celle-là)

Et avec ça, cet attribut pourrait être privé du coup ^^
Comme dis, je ne voulais pas preshot le travaille à effectuer lors du usecase, peut-être qu'il y a d'autres méthodes plus adaptées qui émergeront ^^